sections/featured-product.liquid

```diff
@@ -190,6 +190,7 @@
                 </p>
               {%- when 'quantity_selector' -%}
                 <div
+                  id="Quantity-Form-{{ section.id }}"
                   class="product-form__input product-form__quantity{% if settings.inputs_shadow_vertical_offset != 0 and settings.inputs_shadow_vertical_offset < 0 %} product-form__quantity-top{% endif %}"
                   {{ block.shopify_attributes }}
```

Comment:
Good find! Just checked and this is an existing bug in Dawn `7.0.1` (see video below), I can open an issue for this. I think this wasn't caught because opening the cart drawer on the cart page is an edge case and not often tested.

https://user-images.githubusercontent.com/3710548/214688931-7790080e-4514-4923-9c3c-3cca7daac38b.mp4

Reply:
Thanks for checking! I agree this can definitely be tackled separately plus it is an edge case a well 👍